script.js

```diff
@@ -61,11 +82,61 @@ function matrixDefault()
 function drawArena()
 {
 	gameArena.clearRect(0, 0, width, height);
-	
-	if(turnCount % 2 == 0)
-		gameArena.strokeStyle = "red";
-	else
-		gameArena.strokeStyle = "green";
+
+	if(turnCount === 0)
+	{
+		if (!playersEliminated['red'])
```

Comment:
Instead of passing `'red'`, `'blue'`, etc. you can create another array with all the color names, and just iterate from 0->3. This would easily allow adding more colors later on :slightly_smiling_face: